src/sentry/api/endpoints/organization_missing_org_members.py

```diff
@@ -39,10 +39,10 @@ class MissingMembersPermission(OrganizationPermission):
 def _get_missing_organization_members(
     organization: Organization, provider: str, integration_ids: Sequence[int]
 ) -> QuerySet[WithAnnotations[CommitAuthor]]:
-    member_emails = set(organization.member_set.exclude(email=None).values_list("email", flat=True))
-    member_emails.update(
-        set(organization.member_set.exclude(user_email=None).values_list("user_email", flat=True))
-    )
+    member_emails = set(
+        organization.member_set.exclude(email=None).values_list("email", flat=True)
```

Comment:
oh - does `|` join them?
you could also do `set1.union(set2)` right?

Reply:
or that yes